esy.json

```diff
@@ -18,13 +18,22 @@
   "devDependencies": {
     "@opam/merlin": "^3.2.2",
     "@opam/merlin-lsp": "*",
-    "ocaml": "~4.7.1000"
+    "ocaml": "~4.7.1000",
+    "@opam/httpkit": "*",
```

Comment:
This is a wrong place to add dependencies.

1. It's a monorepo config, so anything that depends on `brisk-macos` will fail to find those dependencies
2. It's `devDependencies` - meaning they are either for tooling like merlin, or some local deps you'd use for tests, they will not be available in runtime

We'll have a detailed design doc and contributing guide, but for now:

1. Add those deps to the `brisk-macos`
2. Also add the resolutions to the relevant example apps (we have just components for now)